package.json

```diff
@@ -3,7 +3,7 @@
   "version": "0.0.1",
   "description": "A tool for tracking blogs in orbit around Seneca's open source involvement",
   "scripts": {
-    "test": "echo \"TODO!\" && exit 1",
+    "test": "eslint --ignore-path .gitignore .",
```

Comment:
A couple things here:

1. Create a separate script for eslint, and then call it from test:

    ```
    "eslint": "eslint --ignore-path .gitignore .",
    "test": "npm run eslint"
    ```
2. Let's also add an eslint-fix script, to automatically fix the obvious/automatically correctable issues in code:
    ```
    "eslint-fix": "eslint --fix --ignore-path .gitignore .",
    ```

Reply:
Should be resolved in new version